endpoints-framework/src/main/java/com/google/api/server/spi/handlers/ApiProxyHandler.java

```diff
@@ -25,17 +25,12 @@
  * Serves the API proxy source with the correct API path.
  */
 public class ApiProxyHandler implements DispatcherHandler<EndpointsContext> {
-  public static final String DEFAULT_API_PATH = "/_ah/api";
-
   private volatile String cachedProxyHtml;
 
   @Override
   public void handle(EndpointsContext context) throws IOException {
     if (cachedProxyHtml == null) {
-      String defaultProxyHtml = IoUtil.readResourceFile(ApiProxyHandler.class, "proxy.html");
-      String apiPath = context.getRequest().getServletPath();
-      cachedProxyHtml = DEFAULT_API_PATH.equals(apiPath) ? defaultProxyHtml
-          : defaultProxyHtml.replace(DEFAULT_API_PATH, apiPath);
+      this.cachedProxyHtml = IoUtil.readResourceFile(ApiProxyHandler.class, "proxy.html");
```

Comment:
other lines in this method use `cachedProxyHtml` without the `this.`. We should definitely be consistent; I think it should be fine to use `cachedProxyHtml` without `this.` here too.